TRANSLATE.rst

```diff
@@ -0,0 +1,52 @@
+******************
+Translation Guide 
+******************
+
+.. contents:: 
+
+You can help to translate the Read the Docs Sphinx Theme.
+
+Languages availables:
+
+- Spanish contributed by Leonardo J. Caballero G.
+- Your Language by YOUR NAME.
+
+Installation
+============
+
+For translate the Read the Docs Sphinx Theme you need install the following packages:
+
+.. code:: bash
+
+    $ pip install babel Jinja2
```

Comment:
@agjohnson I agree with you, but it is not clear when downloading this package so at least I indicate that dependencies minimally require to generate the gettext files with the Babel library from this package. 

Remember that when you install this package using the command **python setup.py install** does not install any dependencies, ie. does not install Sphinx package.